src/main/resources/templates/feed.html

```diff
@@ -29,8 +29,16 @@ <h1 th:text="${entries.size()} + ' entries'"></h1>
         </tbody>
     </table>
 
-    <a th:if="${previousPageLink}" th:href="${previousPageLink}">previousPage</a>
-    <a th:if="${nextPageLink}" th:href="${nextPageLink}">nextPage</a>
+    <li th:if="${pagination.hasPreviousPage()}" class="previous">
```

Comment:
You said that you will ask Robin to look at this so I didn't give much time to look at the html. I am not sure we even need <li>. I copied the pieces from alpha so forgot to copy <ul> part. Although I agree these <li> must be covered in <ul>.